docs/source/resources/package-spec.rst

```diff
@@ -314,7 +314,7 @@ parts:
 Remember that the version specification cannot contain spaces,
 as spaces are used to delimit the package, version, and build
 string in the whole match specification. ``python >= 2.7`` is an
-invalid match specification. However,``"python >= 2.7"`` (with double or single quotes) is
+invalid match specification. However,"python >= 2.7" (with double or single quotes) is
```

Comment:
I am glad they are working now. The issue is because of  space, which I didn't give while putting back-ticks after commas.
As those back-ticks were working on GitHub pages, So they were little tricky for me to Identify. Thank you for your help